website/README.md

```diff
@@ -75,6 +75,10 @@ If you're developing the site, you should know a little bit about Hugo and Docsy
 
 ## Troubleshooting
 
+To fix the Docker Error 255 during the Website launch on Apple Silicon:
```

Comment:
```suggestion
### Docker Error 255 on Apple Silicon

To fix the Docker Error 255 during the Website launch on Apple Silicon:
```

It would be good to match this with the format below